integration_tests/features/Mempool.feature

```diff
@@ -0,0 +1,83 @@
+@propagation
+Feature: Mempool
+
+ 
+  Scenario: Transactions are propagated through a network
+    Given I have 10 seed nodes
+    And I have a base node SENDER connected to all seed nodes
+    And I have 10 base nodes connected to all seed nodes
+    When I wait 100 seconds
```

Comment:
These waits need justifications in comments so that they can be improved upon in future. Describe why the wait is 100 seconds so that a future dev can know where to start optimizing